pig/src/main/java/org/apache/zeppelin/pig/PigQueryInterpreter.java

```diff
@@ -134,9 +135,12 @@ public InterpreterResult interpret(String st, InterpreterContext context) {
           return new InterpreterResult(Code.ERROR, ExceptionUtils.getStackTrace(e));
         }
       }
+      if (e.getCause() instanceof ParseException) {
+        return new InterpreterResult(Code.ERROR, e.getMessage());
```

Comment:
should `e.getMessage()` be `e.getCause().getMessage()`? or `ExceptionUtils.getStackTrace(e)`?